src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,102 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int initialCapacity = 16;
+    private static final double loadFactor = 0.75;
+    private int elementCount;
+    private Cell[] hashMap;
+
+    public MyHashMap() {
+        hashMap = new Cell[initialCapacity];
+    }
+
+    private void resize() {
+        elementCount = 0;
+        Cell<K, V>[] oldHashMap = hashMap;
+        hashMap = new Cell[hashMap.length * 2];
+        for (Cell cell : oldHashMap) {
+            while (cell != null) {
+                put((K) cell.key, (V) cell.getValue());
+                cell = cell.next;
+            }
+        }
+        getSize();
```

Comment:
should we call this method?

Reply:
Oh, no. Just forgot to erase after some troubleshooting